### DISCUSSION TITLE
Can I set printer defaults?

### DISCUSSION THREAD

**Discussion by EllieMelloy** [sumatrapdfreader/sumatrapdf#3528] [General]
Being in the UK I have a Xerox B210 A4 printer and have set printing defaults in Devices>Xerox B210>Printing Preferences. This sets the default for every other software I use but when I print a page from Sumatra it resets to Legal size and I have to change it manually for each document, each time the application opens. It doesn't change the default to Legal, just for that session. If I create an A4 document in another app such as Word, then export to PDF and open in Sumatra, it still defaults to Legal.

Anyone know how I can make it permanently A4?

This is on version 3.4.6 64-bit, WIndows 10 22H2 x64 19045.3031

**Comment by GitHubRulesOK** [comment]
Printing uses defaults
So Primarily for a European its A4 portrait for an American its Portrait Letter. 
Next that's over-ridden by Windows to System Default printer setting 
That in turn may be over-ridden by last used form, (Allow Windows to Manage my Printer).
Next its the specific printer may have user preferred settings within an allowed range.

Usually what SumatraPDF is offered by Windows as Media for initialisation is the current Default Printer. Thus currently I have let sytem "Manage ON" and my last default was print to PDF and that was set to A4 Portrait.

Thus any new print will try to work to that last default set, needing me to switch From A4 to Letter or Legal if so desired.

Thus its ODD if your in UK and Default printer is A4 and set to permanently A4, as to why yours is starting off as Legal, that should be offered default GB / UK metric A4 (unless its a US windows, or Managed by Windows for a different default setting)
